.github/workflows/test-runner-pr-integration.yml

```diff
@@ -81,6 +81,7 @@ jobs:
         cd workspace-manager-clienttests
         echo "Test Runner config"
         ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
+        ./render-k8s-config.sh wsmtest ${{ steps.vault-token-step.outputs.vault-token }}
```

Comment:
Perhaps add a comment here noting that this isn't necessary at the moment, because the line below is running tests against a local WSM server, not one deployed in K8S. I do think this is a good addition though, both for clarity and consistency across GH actions, and also because eventually I think the on-PR GH actions will be run against a deployed WSM so that would mean just updating this.

Reply:
Added comments as suggested.